src/DatasetAPI/Datasets.jl

```diff
@@ -178,6 +178,12 @@ function Base.getproperty(x::Dataset, k::Symbol)
         x[k]
     end
 end
+
+function readcubedata(ds::Dataset)
+    inmemcubes = OrderedDict(key=> readcubedata(val) for (key, val) in pairs(ds.cubes))
+    Dataset(inmemcubes, ds.axes, ds.properties)
+end
```

Comment:
I added it to the readcubedata method for cubes because this would also be applicable to single cubes.

Reply:
But then we also want to show it for larger data cubes. And if your variables are small by themselves it will not be shown

Reply:
ahh!! yes! you are right. Then is fine!

Reply:
I am wondering whether we should include a overall check for the dataset that the combination of all variables is not larger than the max_cache.

Reply:
also true. Otherwise, it will also crash!

Reply:
Done. By the way, this is not crashing but running into an OutOfMemory error, but this might be different, when the data is only slightly larger than max_cache. where it can just fit into RAM but then nothing else will.